### DISCUSSION TITLE
Create a ‘POI exists’ button

### DISCUSSION THREAD

**Discussion by meinrad-0** [organicmaps/organicmaps#2386] [Ideas]
Hi all!
As an outdoor enthousiast I am very happy with OM! I use it especially while hiking, also in remote areas. While in the outdoors, I highly depend on watersources. OM is a great way to find them, however the OSM data is not always very accurate (especially in remote places where not so many people go). So it happens sometimes that a watersource is actually not there anymore. Most of the times I try to verify the watersources by checking other information sources beforehand. However, I think there could be an opportunity to improve OM in this respect.
Now there is an option to delete or add a POI. I think it would be great added value to know when someone passed a POI for the last time and confirmed it is still there. The OSM tag ' check_date ' would probably be appropriate for this.
So this is what I am thinking: press a POI, then it shows 'edit place' (so far nothing new) and below that something like 'confirm that this place is here' which would than at the check_date in OSM and ofcourse show this date in OM.

It is a very rough idea and probably will not work with all POI and I guess there are some issues with OSM implementation, but still I think it could be valuable.

What do you think?

**Comment by biodranik** [comment]
Even by confirming that the poi is still there you can not guarantee that it will disappear tomorrow.

If there is a poi, you add it or don't touch it. If it's absent, you delete it. What are pros of a more complex approach you're offering?

**Reply by meinrad-0** [reply]
First of all, I totally agree with you; if it isnt on the map you add it, if it isnt their on the ground you delete it. But not all OM user use this function (although its a low barrier). So this approach, if implemented in a simple way (only one click), could even reach more OM users. 

But what would it bring then? I think this approach would offer more knowledge to the end user. It adds hugely to the accuracy of the map, in the sense that I **know** that the information is up to date.

Let me give an example (which I come across all of Europe):
I see 3 watersources (eg springs) along the trail. Concluding from OM I do not know more. Concluding from experience I would guess at least one is out of use. If I check the raw data on OSM, I see that 2 have been added last year and one back in 2015. For me, there would be a much higher certainty that the newly added sources are actually there (you are right in that it could be gone a day later, but with springs this more often a matter of years than days). But maybe someone passed the 2015 sources yesterday, but maybe also no one passed it in 7 years. I think it would make a huge difference to know that.

Another case would be with small supermarkets. They go and come very quickly. Now I normally check Google Maps to see when a last review was written, because it indicates that someone was there and the store probably is there.

I think one could also see it as a very fundamental way of reviewing integrated in the OSM environment, on which there is often discussion in the OSM/OM community. It enables to add more up to date information to a POI. As a reviewing option it is very basic, but could grow to something bigger

**Reply by biodranik** [reply]
Storing the info when the feature was created or edited is a big overhead to the data size. It is also not always consistent, because many features are edited from home.

FYI, we want to display and allow users to add/edit OSM `description` field. It's not directly relevant but can help a bit.

The out-of-date problem is fundamental, and it can be solved only if it will be very easy for users to add/edit/remove features. Instead of discussing the "it is still there" button, let's focus on a better UX for "it's not there" or "it changed". It will make the world better sooner )

**Comment by eisams** [comment]
Related, it would be a good idea to show "when was the POI last confirmed to exist" by surfacing check_date and check_date:opening_hours

That would make it more reliable to use the POI information

Also seems like there are related tags with similar information (source:date and survey:date)

https://wiki.openstreetmap.org/wiki/Key:check_date
